composables/useCart.ts

```diff
@@ -0,0 +1,57 @@
+import { useNuxtApp } from "#app"
+import { getCartQuery } from "~~/gql/queries/cart"
+import { addToCartMutation } from "~~/gql/mutations"
+
+export const useCart = () => {
+  const cart = ref({})
```

Comment:
Refs in useCart need to be changed to useState to share the same cart across components

Reply:
Resolved